### ISSUE TITLE
Could not set editor on Linux

### ISSUE THREAD

**Issue by dandevelo** [fman-community/fman#120]
Tried to set Sublime Text as editor on Linux. Got this error:

```
Command 'OpenWithEditor' raised exception.

Traceback (most recent call last):
  File "/opt/fman/Plugins/Core/core/commands.py", line 175, in __call__
    self._open_with_editor(file_to_edit)
  File "/opt/fman/Plugins/Core/core/commands.py", line 199, in _open_with_editor
    save_json('Core Settings.json')
  File "fman/__init__.py", line 141, in save_json
  File "fman/impl/plugins/__init__.py", line 20, in save_json
  File "fman/impl/plugins/config/json_.py", line 23, in save
  File "fman/impl/plugins/config/json_.py", line 100, in write_differential_json
PermissionError: [Errno 13] Permission denied: '/opt/fman/Plugins/Core/Core Settings (Linux).json'
```
Closing the error and hitting F4 again will invoke the editor succesfully. However, the settings are not persisted so next time I start the app it asks me to pick and editor again.

**Comment by mherrmann** [comment]
Some research shows that this bug was introduced in v0.3.5 and affects all OSs :/ It's fixed now. The fix will go live with v0.3.8.

**Comment by mherrmann** [comment]
fman 0.3.8 is live :)
